types/index.ts

```diff
@@ -1,9 +1,11 @@
 export interface Tag {
   _id: string
   name: string
+  // Video count
+  __v: number
```

Comment:
This is a poorly named variable. the comment associated with it proves this

Reply:
Yep. Resolved in: https://github.com/State-Channel-4/frontend/pull/43/commits/303df1ce1847ab510482b75fdcc8d545c95ca35e